packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/static/dwac-21.7.js

```diff
@@ -0,0 +1,476 @@
+                    captureObject(configs[i])
+                }
+            } else {
+                if (configs[0] instanceof Object) {
+                    captureObject(configs[0])
+                } else {
+                    captureObject(configs)
+                }
+            }
+        }
+
+        // Submit captured data if appropriate
+        if (domReady) {
+            submitCollectedData()
+        }
+    }
+
+    // Enhance existing capture function with submission step
+    dw.ac.capture = captureAndSend
+
+    // expose debug API
+    dw.ac.setDebugEnabled = setDebugEnabled
+
+    dw.ac._handleCollectedData = function () {
+        domReady = false
+        dw.ac._events.forEach(captureAndSend)
+        domReady = true
+        submitCollectedData()
+    }
+
+    dw.ac._scheduleDataSubmission = function () {
+        if (dw.ac._submitTimeout) {
+            clearTimeout(dw.ac._submitTimeout)
+        }
+        dw.ac._submitTimeout = setTimeout(dw.ac._handleCollectedData, 500)
+    }
+
+    // Added specifically for PWA kit to set currency for Analytics Context
+    dw.ac._setSiteCurrency = function setSiteCurrency(currency) {
+        analyticsContext.siteCurrency = currency
+    }
+
+    // replace jQuery.ready() function
+    ;(function onDocumentReady(callback) {
+        // Catch cases where $(document).ready() is called after the browser event has already occurred.
+        if (document.readyState === 'complete') {
+            // Handle it asynchronously to allow scripts the opportunity to delay ready
+            setTimeout(callback, 1)
+        }
+
+        // Mozilla, Opera and webkit nightlies currently support this event
+        if (document.addEventListener) {
+            DOMContentLoaded = function () {
+                document.removeEventListener('DOMContentLoaded', DOMContentLoaded, false)
+                callback()
+            }
+            // Use the handy event callback
+            document.addEventListener('DOMContentLoaded', DOMContentLoaded, false)
+            // A fallback to window.onload, that will always work
+            window.addEventListener('load', callback, false)
+            // If IE event model is used
+        } else if (document.attachEvent) {
+            DOMContentLoaded = function () {
+                // Make sure body exists, at least, in case IE gets a little overzealous
+                if (document.readyState === 'complete') {
+                    document.detachEvent('onreadystatechange', DOMContentLoaded)
+                    callback()
+                }
+            }
+            // ensure firing before onload, maybe late but safe also for iframes
+            document.attachEvent('onreadystatechange', DOMContentLoaded)
+
+            // A fallback to window.onload, that will always work
+            window.attachEvent('onload', callback)
+        }
+    })(function () {
+        dw.ac._handleCollectedData()
+    })
+})(window.dw || {})
```

Comment:
@avinashkasipathy why is this file being re-added? Perhaps you're behind `develop` where this file was moved / changed?